site/docs/tasks/custom-taxonomy.md

```diff
@@ -0,0 +1,42 @@
+# Using a Custom Taxonomy for Resource Validation
+
+The project defines a set of immutable structural JSON schemas, or "taxonomies" for resources deployed in Fybrik. 
+However, since the taxonomy is meant to be configurable, a `taxonomy.json` file is referenced from these schemas for any definition that is customizable.
+
+The `taxonomy.json` file is generated from a base taxonomy and zero or more taxonomy layers:
+
```

Comment:
unrelated question - is there a way to comment in taxonomy.json file the base and taxonomy layers files it was generated from so we could easily know what taxonomy we are using?

Reply:
This will require a change to the CLI tool, we need to make an issue for this